clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -582,7 +582,7 @@ private void updateGroupSubscription(Set<String> topics) {
 
         assignmentSnapshot = metadataSnapshot;
 
-        log.debug("Finished assignment for group: {}", assignments);
+        log.info("Finished assignment for group at generation {}: {}", generation().generationId, assignments);
```

Comment:
Do you think it would be useful to also log the generation somewhere for the followers as well, maybe at the beginning of the rebalance in case it "fails" or else when we log that the SyncGroup was unsuccessful eg because group already started another rebalance?

Reply:
I checked the log4j trace and I think we are good covered with `Received successful JoinGroup response`. So I'm just gonna augment the existing entry at the end of the response handler.